ctapipe/coordinates/pointing_corrections.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+
+class PointingCorrection:
+    """
+    Base class for pointing correction schemes
+    """
+
+    def get_camera_trans_matrix(self):
+        """
+        Get dummy transformation matrix, returns diagonal unit array
+
+        Returns
+        -------
+        matrix: affine translate matrix for camera corrections
+        """
+        return np.matrix(np.diagflat(np.ones(3)))
+
+
+class HESSStylePointingCorrection(PointingCorrection):
+    """
+    Class for taking HESS style pointing corrections and producing the transformation
+    matrix required for transforming in the nominal system
+    """
+    def __init__(self, x_trans, y_trans, rotation, scale):
+        """
+
+        Parameters
+        ----------
+        x_trans: float
+            Translation in x-dimension
+        y_trans: float
+            Translation in y-dimension
+        rotation: float
+            Image rotation
+        scale: float
+            Scaling of focal length
+        """
+        self.x_trans, self.y_trans = x_trans, y_trans
+        self.rotation = rotation
+        self.scale = scale
+
+    def get_camera_trans_matrix(self):
```

Comment:
don't use abbreviations: rename to "get_camera_transformation_matrix".  It could be a property or a lazyproperty as well.